app/objects/secondclass/c_link.py

```diff
@@ -104,6 +106,12 @@ def __init__(self, command, paw, ability, status=-3, score=0, jitter=0, cleanup=
         self._pin = pin
         self.output = False
 
+    def __eq__(self, other):
+        if isinstance(other, Link):
```

Comment:
why is this isinstance needed here? Is there anywhere in the code where this is being compared against something that isn't a link?

Reply:
No I don't believe there is any case where this is being compared to anything that isn't a link, I just added it out of habit since I'm overriding the eq. Happy to remove it otherwise.